evaluate.py

```diff
@@ -163,18 +163,18 @@ def main(args: argparse.Namespace) -> None:
     if issubclass(TASK, ClassificationTask):
         val_row = {
             "split": "val",
-            "classification_model": hparams["classification_model"],
-            "learning_rate": hparams["learning_rate"],
-            "weights": hparams["weights"],
-            "loss": hparams["loss"],
+            "model": model.hparams["model"],
```

Comment:
Bad rebase, these should all be `hparams`, not `model.hparams` after my Lightning PR.